Trello.py

```diff
@@ -0,0 +1,96 @@
+import requests
+import os
+import json
+
+"""
+Required for all Trello API Calls
+"""
+headers = {
+    "Accept": "application/json"
+}
+
+trello_params = {
+    'key': os.getenv('TRELLO_KEY'),
+    'token': os.getenv('TRELLO_TOKEN')
+}
+
+"""
+Get Trello Board 
+"""
+def get_board():
```

Comment:
I can't see this function used anywhere - if that's the expected case then we can get rid of it

Reply:
It wasn't being used as I was in the process of setting this up. I have now completed this function and added it to my render_template